docs/components/index.md

```diff
@@ -67,18 +67,21 @@ In the following example, the `ParentComponent` sets the value of the `Title` pr
 
 ## Child content
 
-Components can set the content of an element in another component by assigning to a property that represents a [RenderFragment](/api/Microsoft.AspNetCore.Blazor.RenderFragment.html) type.
-
-In the following example, the `ChildContent` element of the `ParentComponent` sets the value of the `ChildComponent`'s `ChildContent` property. `ChildContent` is a `RenderFragment` type and supplies the content to the Bootstrap panel.
+Components can set the content of an element in another component. The assigning component provides the content in the tags that specify the receiving component. For example, a `ParentComponent` can provide content that is to be rendered by a `ChildComponent` by placing the content inside **\<ChildComponent>** tags.
 
 *ParentComponent.cshtml*:
 
 [!code-cshtml[](common/samples/2.x/ComponentsSample/Pages/ParentComponent.cshtml?start=1&end=7&highlight=6)]
 
+The receiving component has a `ChildContent` property that represents a [RenderFragment](/api/Microsoft.AspNetCore.Blazor.RenderFragment.html) type. The value of `ChildContent` is positioned in the markup where the content should be rendered.
```

Comment:
The parent component captures child content by defining a property named `ChildContent` of type [RenderFragment](/api/Microsoft.AspNetCore.Blazor.RenderFragment.html). The parent component can then specify `@ChildContent` to render the child content as part of its own rendering.

Reply:
We'll need to be very careful how we handle this.

> The parent component can then specify @ChildContent to render the child content as part of its own rendering.

The parent doesn't specify `@ChildContent` directly (I mean not in its markup by name, "ChildContent") ... it *appears* (just *looks like*) that the child does (in its property and its markup I mean).

If we try to explain it the way that you wrote it, I think we'll have a lot of head scratching.

Another phrase that concerns me ...

> parent component captures child content

... but "child content" can be confused: I don't want a reader to misconstrue that it's the entire rendered child in the parent.

Anyway ... back to my main point ... The "child" is the only place the dev will see an explicit reference to `ChildContent`/`@ChildContent`, so saying that the parent component specifies "ChildContent" is tricky to do without losing the reader. What I suggest is wording it more along the lines of the parent specifies the value of `ChildContent` in the value of the tags that specify the child component ... along those lines. I just want to make it really clear where `ChildContent` is and how it's set by the content of the tags in the parent.